core/frontend/src/components/video-manager/VideoStreamCreationDialog.vue

```diff
@@ -57,7 +57,7 @@
             <v-text-field
               v-model="stream_endpoints[index]"
               label="Stream endpoint"
-              placeholder="e.g. udp://192.168.2.2:5600 or rtsp://0.0.0.0:8554/video0"
+              placeholder="e.g. udp://192.168.2.1:5600 or rtsp://192.168.2.2:8554/video0"
```

Comment:
As you're making this major cleaning (👁️), would you mind declaring this address in some separate file and importing from it where it's been used? That would make sure we're not pointing to different places around the code.

Reply:
major cleaning ¬¬

I can take the placeholder out as it is no longer necessary...